src/test/java/com/nvidia/spark/rapids/jni/HashTest.java

```diff
@@ -261,4 +261,131 @@ void testSpark32BitMurmur3HashListsAndNestedLists() {
       assertColumnsAreEqual(nestedExpected, nestedResult);
     }
   }
+
+  @Test
+  void testXXHash64Strings() {
+    try (ColumnVector v0 = ColumnVector.fromStrings(
+           "a", "B\nc",  "dE\"\u0100\t\u0101 \ud720\ud721\\Fg2\'",
+           "A very long (greater than 128 bytes/char string) to test a multi hash-step data point " +
+           "in the MD5 hash function. This string needed to be longer.A 60 character string to " +
+           "test MD5's message padding algorithm",
+           "hiJ\ud720\ud721\ud720\ud721", null);
+         ColumnVector result = Hash.xxhash64(new ColumnVector[]{v0});
+         ColumnVector expected = ColumnVector.fromBoxedLongs(-8582455328737087284L, 2221214721321197934L, 5798966295358745941L, -4834097201550955483L, -3782648123388245694L, Hash.DEFAULT_XXHASH64_SEED)) {
```

Comment:
Is there a reason we don't use the CPU to hash the same ColumnVector and compare?

Reply:
Not particularly.  Just following what the murmur hash tests do.